collections/_article/godot-xr-update-feb-2025.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Godot XR update - February 2025"
+excerpt: "New plugin releases for Godot XR."
+categories: ["progress-report"]
+author: Bastiaan Olij
+image: /storage/blog/covers/february-2025-xr-progress-update.webp
+image_caption_description: "TBD"
+date: 2025-02-15 12:00:00
+---
+
+## Announcing the Meta Toolkit extension
+
+We are pleased to announce the release of the [Godot Meta Toolkit](https://github.com/godot-sdk-integrations/godot-meta-toolkit), a GDExtension plugin that exposes Meta's [Platform SDK](https://developers.meta.com/horizon/documentation/native/ps-platform-intro) and provides other tools to simplify and accelerate XR development on Meta's platform.
+
+The project is Open Source (MIT license) and maintained by [W4 Games](https://www.w4games.com/) with sponsorship from Meta. Contributions from the community are welcome! 🙂
+
+You can download version 1.0.1 [from GitHub](https://github.com/godot-sdk-integrations/godot-meta-toolkit/releases/tag/1.0.1-stable) or the [Asset Library](https://godotengine.org/asset-library/asset/3673).
```

Comment:
Sorry to do this to you again @BastiaanOlij, but we've just released version 1.0.2:

```suggestion
You can download version 1.0.2 [from GitHub](https://github.com/godot-sdk-integrations/godot-meta-toolkit/releases/tag/1.0.2-stable) or the [Asset Library](https://godotengine.org/asset-library/asset/3673).
```

Hopefully, we won't have another update before this is published :-)

Reply:
Stop working hard!